lib/koala/api.rb

```diff
@@ -61,7 +61,7 @@ def api(path, args = {}, verb = "get", options = {}, &error_checking_block)
         end
 
         # Translate any arrays in the params into comma-separated strings
-        args = sanitize_request_parameters(args)
+        args = sanitize_request_parameters(args) unless Koala.configuration.allow_array_parameters
```

Comment:
Also there may be other things that are later added to `sanitize_request_parameters` that have nothing to do with array serialization, so it might make more sense to break the array serialization out into its own method (thus separating it from general sanitization).